tests/test_git-abort.py

```diff
@@ -0,0 +1,60 @@
+import os
```

Comment:
`test_git_abort.py` would be a better name

Reply:
I juse think `test a command`, so `test git-abort` is `test_git-abort`. And `test the command` should be `test git abort`, it is a noum, but i think both are ok. i will reset it later.